yt_dlp/extractor/niconicochannelplus.py

```diff
@@ -33,10 +33,17 @@ def _find_fanclub_site_id(self, channel_name):
             raise ExtractorError(f'Channel {channel_name} does not exist', expected=True)
         return fanclub_id
 
-    def _get_channel_info(self, fanclub_site_id):
+    def _get_channel_base_info(self, fanclub_site_id):
         return self._call_api(
             f'fanclub_sites/{fanclub_site_id}/page_base_info', item_id=f'fanclub_sites/{fanclub_site_id}',
-            note='Fetching channel info', errnote='Unable to fetch channel info',
+            note='Fetching channel base info', errnote='Unable to fetch channel base info',
+        )['data']['fanclub_site']
+
+    def _get_channel_user_info(self, fanclub_site_id):
+        return self._call_api(
+            f'fanclub_sites/{fanclub_site_id}/user_info', item_id=f'fanclub_sites/{fanclub_site_id}',
+            note='Fetching channel user info', errnote='Unable to fetch channel user info',
+            data=json.dumps('null').encode('ascii'),
         )['data']['fanclub_site']
```

Comment:
```py
    def _get_channel_info(self, fanclub_site_id, endpoint=page_base_info, data=None):
        return self._call_api(
            f'fanclub_sites/{fanclub_site_id}/{endpoint}', item_id=f'fanclub_sites/{fanclub_site_id}',
            data=data, note='Fetching channel base info', errnote='Unable to fetch channel base info',
        )['data']['fanclub_site']
```
is much simpler imo

You can call it as age_limit = `traverse_obj(self._get_channel_info(fanclub_site_id, 'user_info', b'"null"'), ...)`

Reply:
Oops, I reviewed just the new diff. Ignore this